main.py

```diff
@@ -13,21 +14,51 @@ def index():
 
 @app.route("/search", methods=["GET"])
 def search():
-    app.logger.info(f"search(): GET {request.full_path}.")
-    keyword = request.args.get("query", None)
-    if keyword:
-        context_dict = {
-            "subtitle": f"Search Result for {keyword}",
-            "keyword": keyword,
-            "message": None
-        }
+    app.logger.info(f"search(): GET /{request.full_path}")
```

Comment:
`{request.full_path}` だけで、`/search?query=kindle` 的に出力します
```suggestion
    app.logger.info(f"search(): GET {request.full_path}") 
```